compiler/src/iree/compiler/Codegen/Common/EncodingUtils.cpp

```diff
@@ -62,16 +63,18 @@ MaterializeEncodingConversionTarget::MaterializeEncodingConversionTarget(
   // Mark any operation that has operands/results with encoding as
   // illegal.
   markUnknownOpDynamicallyLegal([](Operation *op) {
-    auto typeHasEncoding = [](Type t) -> bool {
+    auto typeHasDataTilingEncoding = [](Type t) -> bool {
       auto tensorType = dyn_cast<RankedTensorType>(t);
-      return tensorType && tensorType.getEncoding();
+      if (!tensorType)
+        return false;
+      return getEncodingAttr(tensorType) != nullptr;
     };
     auto valueHasEncoding = [=](Value v) -> bool {
-      return typeHasEncoding(v.getType());
+      return typeHasDataTilingEncoding(v.getType());
     };
```

Comment:
Given that we rename `*HasEncoding` to `*HasDataTilingEncoding`, I think it makes sense to update the `valueHasEncoding` name as well for consistency.